nextflow_schema.json

```diff
@@ -125,11 +125,11 @@
                     "default": true,
                     "help_text": "Turn on the merging of read-pairs of paired-end short read sequencing data for AdapterRemoval (this is performed automatically with fastp).\n\n> Modifies tool parameter(s):\n> - AdapterRemoval: `--collapse`\n"
                 },
-                "shortread_qc_excludeunmerged": {
+                "shortread_qc_includeunmerged": {
                     "type": "boolean",
                     "fa_icon": "far fa-times-circle",
-                    "description": "Discard unmerged reads from paired-end merging",
-                    "help_text": "Turns off the inclusion of unmerged reads in resulting processing FASTQ file of paired-end sequencing data when using `fastp`.\n\nThis can be useful in cases where you prefer to have very short reads (e.g. aDNA), thus excluding longer-reads or possibly faulty reads where one of the pair was discarded.\n\n> Modifies tool parameter(s):\n> - removed from reads `--include_unmerged`\n"
+                    "description": "Include unmerged reads from paired-end merging in the downstream processing",
+                    "help_text": "Turns on the inclusion of unmerged reads in resulting processing FASTQ file of paired-end sequencing data when using `fastp`.\n\nExcluding unmerged reads can be useful in cases where you prefer to have very short reads (e.g. aDNA), thus excluding longer-reads or possibly faulty reads where one of the pair was discarded.\n\n> Modifies tool parameter(s):\n> - adds option `--include_unmerged`\n"
```

Comment:
Should mention this applies to AdapterRemoval too (influencing what output files are mixed into the final output FASTQ).

The end help text here should be something like:

> Modifies fastp parameter(s): `--<whatever it is>`

Reply:
I've added some words on AdapterRemoval. Let me know what you think.